client/src/Widgets/ActivityTable.tsx

```diff
@@ -20,15 +20,210 @@ import {
+      ? activity.assignmentStatus
+      : "";
+
+  if (
+    activity.assignmentStatus === "Open" &&
+    activity.closeTime !== undefined
+  ) {
+    assignmentStatusString =
+      assignmentStatusString + " until " + activity.closeTime;
+  }
+
+  let tooltipLabelString: string = activity.isFolder
+    ? activity.isPublic || activity.isShared
+      ? "Folder / Public"
+      : "Folder / Private"
+    : activity.authorRow
+      ? activity.ownerName
+      : "Activity" +
+        (activity.assignmentStatus ? " / " + activity.assignmentStatus : "");
+
+  function saveUpdatedTitle() {
+    if (rowTitle !== activity.title && activity.id !== undefined) {
+      fetcher.submit(
+        {
+          _action: "update title",
+          id: activity.id,
+          cardTitle: rowTitle,
+          isFolder: Boolean(activity.isFolder),
+        },
+        { method: "post" },
+      );
+    }
+  }
+
+  return (
+    <LinkBox
+      as={Tr}
+      key={(activity.authorRow ? "author" : "activity") + activity.id}
+      data-test="Activity Link"
+      cursor="pointer"
+      _hover={{ backgroundColor: "#eeeeee" }}
+      borderBottom="2px solid gray"
+      width="100%"
+    >
+      <Td p="0" m="0" width="20px">
+        <LinkOverlay href={activity.cardLink ? activity.cardLink : ""}>
+          <Tooltip label={tooltipLabelString}>
+            <Box m="0" p="0">
+              {activity.authorRow ? (
+                <Avatar size="sm" name={activity.ownerName} marginLeft="1em" />
+              ) : (
+                <Icon
+                  as={
+                    activity.isFolder
+                      ? FaFolder
+                      : activity.assignmentStatus === "Unassigned"
+                        ? RiDraftFill
+                        : MdAssignment
+                  }
+                  color={
+                    activity.isFolder
+                      ? "#e6b800"
+                      : activity.assignmentStatus === "Unassigned"
+                        ? "#ff6600"
+                        : activity.assignmentStatus === "Open"
+                          ? "#009933"
+                          : "#cc3399"
+                  }
+                  boxSize={12}
+                  paddingLeft="1em"
+                />
+              )}
+            </Box>
+          </Tooltip>
+        </LinkOverlay>
+      </Td>
+      <Td>
+        <HStack>
+          <Editable
+            value={rowTitle}
+            data-test="Editable Title"
+            zIndex="1"
+            startWithEditView={activity.autoFocusTitle}
+            isDisabled={!activity.editableTitle}
+            onChange={(txt) => setRowTitle(txt)}
+            onSubmit={saveUpdatedTitle}
+          >
+            <EditablePreview
+              cursor={activity.editableTitle ? "auto" : "pointer"}
+              maxHeight="1.5em"
+            />
+            <EditableInput
+              maxLength={191}
+              onBlur={() => {
```

Comment:
Now that we got keyboard navigation work, a flaw from this is revealed. If you tab off this title, the `onBlur()` is activated, adding this listener. This means that the next click (which is the next pressing Enter on the keyboard) is ignored, created strange behavior. For example, after tabbing away from a title, one has to press Enter twice on a folder/activity icon to navigate to that item!

Reply:
This is the best workaround I could come up with, to just remove the listener after 200 ms. I don't know why there isn't a way to distinguish blur via mouse versus blur via keyboard.
```
onBlur={() => {
  // prevent click default/propagation behavior one time (aka right now as user is clicking to blur input)
  const clickListener = (e) => {
    e.preventDefault();
    e.stopPropagation();
  };
  document.addEventListener("click", clickListener, {
    capture: true,
    once: true,
  });
  setTimeout(() => {
    document.removeEventListener("click", clickListener, {
      capture: true,
    });
  }, 200);
}}
```

Reply:
If you don't want to use the timeout, I discovered that adding an event listener on "keyup" rather than "keydown" does work, as the "keyup" event is triggered after the "blur" event when tabbing out of the input.
```
document.addEventListener(
  "keyup",
  () => {
    document.removeEventListener("click", clickListener, {
      capture: true,
    });
  },
  { once: true },
);
```